lib/fontist/font.rb

```diff
@@ -50,6 +56,60 @@ def check_or_create_fontist_path!
       end
     end
 
+    def missing_fonts(existing_paths)
+      available = available_fonts || []
+      existing = filenames(existing_paths)
+      available - existing
+    end
+
+    def available_fonts
+      available_by_style || available_by_font || available_by_formula
+    end
+
+    def available_by_style
+      return unless @style
```

Comment:
> How about shortcutting it with `return unless formula`?

This comment is from https://github.com/fontist/fontist/pull/148#discussion_r498679481